storm-server/src/main/java/org/apache/storm/nimbus/LeaderListenerCallback.java

```diff
@@ -60,31 +62,35 @@
     private final TopoCache tc;
     private final IStormClusterState clusterState;
     private final CuratorFramework zk;
-    private final LeaderLatch leaderLatch;
+    private final ILeaderElector leaderElector;
     private final Map conf;
     private final List<ACL> acls;
+    private final int requeueDelayMs;
 
     /**
      * Constructor for {@LeaderListenerCallback}.
      * @param conf config
      * @param zk zookeeper CuratorFramework client
-     * @param leaderLatch LeaderLatch
+     * @param leaderElector Leader elector
      * @param blobStore BlobStore
      * @param tc TopoCache
      * @param clusterState IStormClusterState
      * @param acls zookeeper acls
      */
-    public LeaderListenerCallback(Map conf, CuratorFramework zk, LeaderLatch leaderLatch, BlobStore blobStore,
+    public LeaderListenerCallback(Map conf, CuratorFramework zk, BlobStore blobStore, ILeaderElector leaderElector,
```

Comment:
nit: param orders changed while it's not reflected to javadoc

Reply:
Fixed.